docs/synapse/userguides/model_update_v2_176_0.rst

```diff
@@ -0,0 +1,27 @@
+.. _userguide_model_v2_176_0:
+
+v2.176.0 Model Updates
+######################
+
+The following model updates were made during the ``v2.176.0`` Synapse release.
+
+**New Forms**
```

Comment:
Should these be in the same format as the v2.177.0 updates or is that something to be cleaned up later?

Reply:
I want to clean them up later.